integration.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Compiling the webserver
+echo "Building webserver."
+make
+
+# starting webserver
+{	
+	echo "Starting webserver."
+	timeout 10 ./webserver test_config_1
+	echo "Stopping webserver."
+}&
+sleep 3
+
+# testing webserver
+echo "Running command curl -D headers.txt http://localhost:12345 > response.txt"
+curl -D headers.txt http://localhost:12345 > response.txt
+
+# checking reponse
+echo "Checking response."
+if grep -q "HTTP/1.1 200 OK" headers.txt;then
+	if grep -q "Content-Type: text/plain" headers.txt;then
+		#Checking if content-length is correct
+		count=`wc -c response.txt | awk '{print $1}'`
+		if grep -q "Content-Length: $count" headers.txt;then
```

Comment:
checks look good, could also add a static response file containing the expected output and check for file equality to guarantee check filters all possible errors also in transmission.. but could be a portability issue so just an option to be thorough